glide.yaml

```diff
@@ -0,0 +1,24 @@
+package: github.com/kubernetes-incubator/rktlet
+import:
+- package: k8s.io/kubernetes
```

Comment:
@yifan-gu you can simply pin to a commit hash in `glide.yaml` like we do for grpc in rkt, see https://github.com/coreos/rkt/blob/master/glide.yaml#L257. Just add `version: deadbeaf...`.